testing/yuca/ca/test_cca.py

```diff
@@ -4,19 +4,50 @@
 
 import torch
 
-from yuca.multiverse import CA
+from yuca.ca.continuous import CCA
 
 
-class TestCA(unittest.TestCase):
+class TestCCA(unittest.TestCase):
 
     def setUp(self):
         pass
 
+    def test_get_frame(self):
+
+        ca = CCA()
+        ca.default_init()
+
+        x = torch.randn(1, 1, 32, 32)
+        for frame_mode in [0,1,2,3,4]:
+            frame_return = ca.get_frame(x,mode=frame_mode)
+
+            if frame_mode == 0:
+                self.assertEqual(1, len(frame_return))
+            elif True in (frame_mode == np.array([1,2])):
+                self.assertEqual(2, len(frame_return))
+            elif True in (frame_mode == np.array([3])):
+                self.assertEqual(3, len(frame_return))
+            elif True in (frame_mode == np.array([4])):
+                self.assertEqual(4, len(frame_return))
+            
+    def test_alive_maxk(self):
+
+        ca = CCA(external_channels=4)
+        ca.default_init()
+
+        x = torch.randn(1, 4, 32, 32)
+        masked = ca(x)
+
+        
+
     def test_multiverse_forward(self):
 
-        for mode in ["functional", "neurofunctional", "neural"]:
-            ca = CA(ca_mode = mode)
-            ca.random_init()
+        for mode in ["functional", "neurofunctional"]:
```

Comment:
Neural functionality moved to `NCA` class. However, `ca_modes` should also be cleaned up before opening a PR from apprentice to master.